src/middlewared/middlewared/plugins/failover_/reboot.py

```diff
@@ -0,0 +1,58 @@
+import asyncio
+import errno
+import time
+
+from middlewared.schema import accepts, returns
+from middlewared.service import CallError, job, Service
+
+
+class FailoverService(Service):
+
+    class Config:
+        cli_private = True
+        role_prefix = 'FAILOVER'
+
+    @accepts()
+    @returns()
+    @job(lock='reboot_standby')
+    async def reboot_standby(self, job):
```

Comment:
This method provides a loaded foot-gun and allows someone to log into the standby and then reboot the active. So naming it `reboot_standby` is wrong. This is why I added a method named `become_passive`.